tools/windows-node-installer/pkg/cloudprovider/aws/ec2.go

```diff
@@ -146,6 +150,7 @@ func (a *AwsProvider) CreateWindowsVM() (credentials *types.Credentials, rerr er
         $file = "$env:temp\ConfigureRemotingForAnsible.ps1"
         (New-Object -TypeName System.Net.WebClient).DownloadFile($url,  $file)
         powershell.exe -ExecutionPolicy ByPass -File $file
+		powershell -NonInteractive -ExecutionPolicy Bypass Add-WindowsCapability -Online -Name OpenSSH.Server~~~~0.0.1.0
```

Comment:
For some reason, this is looking properly aligned locally for me.

Reply:
Enable whitspace display on your IDE and that might show you why.

Please update the comment for `userDataWinrm` variable.